api/api.py

```diff
@@ -0,0 +1,109 @@
+import pickle
+from typing import List
+from .exceptions import InvalidModelError
+
+from sentence_transformers import SentenceTransformer
+
+
+class Ner4Opt(object):
+
+    CLASSICAL = 'classical'
```

Comment:
On one hand, I like that it matches the paper.
On the other hand, another option could be Lexical, Lexical+, Semantic, Hybrid?